Gemfile

```diff
@@ -2,6 +2,8 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.5.5'
+gem 'bcrypt'
+gem 'dotenv-rails', groups: [:development, :test]
```

Comment:
Not a big deal, but you have a gem group of `:development` and `:test` in line [32](https://github.com/tech-lady/Mediaum-api/pull/3/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R32), why not move this gem within the block and remove the "group" option